src/rules/each-key-value-single-line/README.md

```diff
@@ -0,0 +1,48 @@
+# each-key-value-single-line
+
+ Disallow situations where users loop over map-keys and fetch the value inside the loop.
+
+ ```scss
+ $font-weights: ("regular": 400, "medium": 500, "bold": 700);
+ @each $key in map-keys($font-weights) {
+   $value: map-get($font-weights, $key);
+/**        ↑
+ * This call should be consolidated into the @each call.
+ }
+  }
```

Comment:
The syntax inside this SCSS block is broken, there is an extra `}` and the comment is not closed.